### ISSUE TITLE
SWC Title Change Suggestions

### ISSUE THREAD

**Issue by b-mueller** [SmartContractSecurity/SWC-registry#63]
I feel like the following changes would be appropriate:

*SWC 121 - "Signature Replay" -> "Replay Attack Vulnerability"*

To make it more in line with the rest of the SWC titles ("Signature Replay" is an attack technique and not a weakness / vulnerability).

*SWC 122 - "Pseudo Signature Verification based on msg.sender"  -> "Broken Signature Verification"*

Again, to make it more broadly refer to cases where signature verification is implemented wrongly and can be bypassed.

**Comment by thec00n** [comment]
> Signature Replay" is an attack technique and not a weakness / vulnerability

I agree. How about "Missing protection against Signature Replay Attacks"? This would refer to the weakness. 

> SWC 122 - "Pseudo Signature Verification based on msg.sender" -> "Broken Signature Verification"

The SWC is a collection of variants that tie into CWE Base and Class types and being specific and not too broad is a good. I am proposing to rename this to "Pseudo Signature Verification" though to make it a little broader and fit this to more test cases. The variant refers to not to wrongly implemented signature verification methods that if implemented correctly would work but to verification methods that by design can never provide the signature verification properties from a proper implementation.

**Comment by b-mueller** [comment]
I agree but "Pseudo" isn't a good wording here. It's a Germanism /
Austrianism and while its meaning is similar it doesn't sound right in this
particular context.

"Broken", "Improperly implemented', ... would be more correct.

**Comment by thec00n** [comment]
> SWC-121: Missing Protection against Signature Replay Attacks
> SWC-122: Improper Signature Verification 

Ok how about this?

**Comment by b-mueller** [comment]
Kewl

**Comment by thec00n** [comment]
#65
